src/main/java/fr/xephi/authme/AuthMe.java

```diff
@@ -1,782 +1,781 @@
-package fr.xephi.authme;

```

Comment:
Sorry, the git client on my mobile is crazy -_- I changed only some wrong console strings...